internal/k8s/ako_init.go

```diff
@@ -573,27 +570,35 @@ func (c *AviController) InitController(informers K8sinformers, registeredInforme
 			utils.AviLog.Warnf("Full sync interval set to 0, will not run full sync")
 		}
 
-		if ctrlAuthToken, ok := utils.SharedCtrlProp().AviCacheGet(utils.ENV_CTRL_AUTHTOKEN); ok && ctrlAuthToken != nil && ctrlAuthToken.(string) != "" {
-			tokenWorker = utils.NewFullSyncThread(time.Duration(utils.RefreshAuthTokenInterval) * time.Hour)
-			tokenWorker.SyncFunction = c.RefreshAuthToken
-			go tokenWorker.Run()
-		}
 	}
 
-	// Leader election happens after populating controller cache and fullsynck8s.
 	ctx, cancel := context.WithCancel(context.Background())
-	leaderElector, err := utils.NewLeaderElector(informers.Cs, c.OnStartedLeading, c.OnStoppedLeading, c.OnNewLeader)
-	if err != nil {
-		utils.AviLog.Fatalf("Leader election failed with error %v, shutting down AKO.", err)
-	}
+	if lib.IsWCP() {
+		lib.AKOControlConfig().SetIsLeaderFlag(true)
+		c.cleanupStaleVSes()
```

Comment:
We have to add the following also here.

```
	// once the l3 cache is populated, we can call the updatestatus functions from here
	restlayer := rest.NewRestOperations(avicache.SharedAviObjCache(), avicache.SharedAVIClients())
	restlayer.SyncObjectStatuses()
```

Reply:
@abhishekbsingh Please take care of this in the upcoming PRs. I am marking this as resolved for now.